### DISCUSSION TITLE
What is the scope of your work?

### DISCUSSION THREAD

**Discussion by ghost** [mdarikrayhan/BAUET_Result_System#5] [Q&A] [answered]
I would like to know about the scope of your work.

**Comment by mdarikrayhan** [comment, marked as answer]
We plan to develop this project by using this will reduce the need for human resources and save a lot of time. The purpose of the BAUET Result system is to automate the existing manual system with the help of computerized equipment and full-fledged computer software, fulfilling their requirements valuable data information can be stored for a longer period with easy accessing and manipulation of the same. The required software and hardware are easily available and easy to work with. Student Result System, as described above, can lead to an error-free, secure, reliable and fast system. It can assist the user to concentrate on their other activities rather than on record keeping. Thus, it will help the organization in better utilization of resources. The organization can maintain computerized records without redundant entries. That means that one need not be distracted by information. Student Result web System, as described above, can lead to an error-free, secure, reliable and fast management system. It can assist the user to concentrate on them rather than on record keeping. Thus, it will help the organization in better utilization of resources. The organization can maintain computerized records without redundant entries. That means that one need not be distracted by information that is not relevant while being able to reach the information. The main objective was to enhance and automate the management and declaration of students’ results using a computerized system.  A well-defined, efficient, controlled and managed information system or software based on web technology storing, processing and providing information through the internet.
